### PR TITLE
Bump prometheus to 2.37.9 to fix CVE 2022 41717

### PR DESCRIPTION
**Pull request by Sumynwa**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./LICENSES-AND-NOTICES/SPECS/data/licenses.json`, `./LICENSES-AND-NOTICES/SPECS/LICENSES-MAP.md`, `./LICENSES-AND-NOTICES/SPECS/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 
Bump to version 2.37.9 to fix CVE-2022-41717
Reference:
https://github.com/prometheus/prometheus/pull/11690
https://go-review.googlesource.com/c/net/+/455635
https://go-review.googlesource.com/c/go/+/455717

###### Change Log  
- Bump package version

###### Does this affect the toolchain?  
NO

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2022-41717

###### Test Methodology

- Pipeline build id: [Buddy Build](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=655716&view=results)

### CODE REVIEW

File: SPECS/prometheus/prometheus.signatures.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
  "Signatures": {
-  "prometheus-2.37.0.tar.gz": "98892e82b97004a458e81f03d804859d485323af2d85c34f8a996e25fe1305a9",
+  "prometheus-2.37.9.tar.gz": "f26eba405e0836c5a53bfff91b45dc71b14900d5edc0fe8db7238d3c85ac45fb",
   "prometheus.conf": "ce522e82dfb2945c520b482b15b5cf591364f7a571f0f28259b64dbeda42b043",
   "prometheus.logrotate": "061b92500cd40fcaaf486ff488bcf1b09eac6743d8e840ba6966dc70d4e2067b",
   "prometheus.service": "29bf1c886e1d55080e859f2afe112bb7344490e6992e946efe3360fd94d1a604",
   "prometheus.sysconfig": "ec89a45641e3411478794106246aa91e7b72f86070a28a4782e3b8be955e4587",
   "prometheus.yml": "0112e0bf54660c5e2391fff11a56404a25684c588caa7281677f7f8e19da6f28",
-  "promu-0.13.0.tar.gz": "3473b87214968c79158f553228baef6e9a37ed3e11e1a4f3e7267ffd3180a8b6"
+  "promu-0.14.0.tar.gz": "1e07b4a6d7f00cb96b3d4c4d8ce3d463501cb0a0f790aed32a87985ce63dab5d"
```
Comment (PawelWMS) [inline SPECS/prometheus/prometheus.signatures.json:9]: Looks like there's a bug in the publishing pipeline. This file was already in our blobstore but with a different hash. We've restored it to fix the 3.0 builds, which started failing after the upload of this one. Please use this hash instead and it should unblock you as well:
```suggestion
  "promu-0.14.0.tar.gz": "d71d2a0d54093f3f17dc406d7a5825b6d6acd304cd90d9c60ed3f1335fb6ed2a"
```

Comment (PawelWMS) [reply SPECS/prometheus/prometheus.signatures.json:9]: Also, please follow [these steps to upload vendored tarballs](https://dev.azure.com/mariner-org/mariner/_wiki/wikis/mariner.wiki/1284/Source-Tar-Publishing?anchor=upload-vendored-source-tarball%3A), so we don't create new blobstores or other file servers, which may be hard to track or secure.

Comment (Sumynwa) [reply SPECS/prometheus/prometheus.signatures.json:9]: Thanks @PawelWMS. Sure.

**Review by jslobodzian — COMMENTED** [review]
Does this conflict or need to be coordinated with: https://github.com/microsoft/azurelinux/pull/10673

**Review by jslobodzian — APPROVED** [review]
please retarget at fasttrack 2.0

**Comment by bhagyapathak** [comment]
Retargeting following PR for fasttrack/2.0 
1. https://github.com/microsoft/azurelinux/pull/10685
2. https://github.com/microsoft/azurelinux/pull/10673

under new PR https://github.com/microsoft/azurelinux/pull/10730

**Pull request closed without merging**